gateway/gateway.proto

```diff
@@ -58,6 +58,9 @@ message EndorseRequest {
     string channel_id = 2;
     // The signed proposal ready for endorsement.
     protos.SignedProposal proposed_transaction = 3;
+    // If targeting the peers of specific organizations (e.g. for private data scenarios),
+    // the list of organizations should be supplied here.
+    repeated string endorsing_organizations = 4;
```

Comment:
Probably ok, but let's design the collection stuff as well to make sure. For example you'd likely pass either the specific organizations, or the collection names.
Also there are some other scenarios like chaincode to chaincode and collections with no private reads that will need to be designed for in the endorsers query, see the various options at https://hyperledger-fabric.readthedocs.io/en/latest/discovery-cli.html#endorsers-query

Reply:
Per discussion, let's proceed as-is for now, we can extend the design for passing collections once that story is developed.

Reply:
the comment should probably mention that the org MSPIDs should be specified, but that can be added later.